src/core/crypto.c

```diff
@@ -343,7 +343,7 @@ QuicCryptoInitializeTls(
 
     Crypto->ResumptionTicket = NULL; // Owned by TLS now.
     Crypto->ResumptionTicketLength = 0;
-    QuicCryptoProcessData(Crypto, !IsServer);
+    (void)QuicCryptoProcessData(Crypto, !IsServer);
```

Comment:
Same ignoring return value question here. If this is valid, maybe a comment explaining why it's getting ignored. Usually I see something like this as a code smell.